akka-cluster-tools/src/main/resources/reference.conf

```diff
@@ -141,6 +141,12 @@ akka.cluster.singleton {
   # the previous oldest confirms that the hand over has started or the previous 
   # oldest member is removed from the cluster (+ akka.cluster.down-removal-margin).
   hand-over-retry-interval = 1s
+  
+  # The number of retries are derived from hand-over-retry-interval and
+  # akka.cluster.down-removal-margin (or ClusterSingletonManagerSettings.removalMargin),
+  # but if removal margin is disabled (or short) the minimum number of retries
```

Comment:
strictly speaking the *minimum* number of retries is always defined by this property.